Task2 Objects and Functions completed

saimaanam opened the pull request:
This PR is for the completion of Task 2 of learning JS- **_Objects and Functions._**

saimaanam committed:
Task2 Objects and Functions completed

saimaanam committed:
Solution for prime number added

DPrakapuk reviewed (changes requested):
//Write a function that determines whether a number is prime or not
1 is not prime, but you function console that is prime 

do not use var, and replcase let on const

saimaanam committed:
Update ObjectsAndFunctions.js

'Number is prime or not' code is reimplemented 
Error for number 1 as Prime number is now sorted.
